ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/AcidState.hs

```diff
@@ -253,9 +356,10 @@ createPrefiltered :: forall p e.
                   -> Map HdAccountId AccountName
                   -> Update' HdWallets e ()
 createPrefiltered mkPrefilteredUtxo accApplyP narrowP addrApplyP pByAccount accountNames =
-    forM_ (Map.toList pByAccount) $ \(accId, p) ->
-        createAccPrefiltered mkPrefilteredUtxo accApplyP narrowP addrApplyP accId p
+    forM_ (Map.toList pByAccount) $ \(accId, p) -> do
+        _ <- createAccPrefiltered mkPrefilteredUtxo accApplyP narrowP addrApplyP accId p
```

Comment:
That's most likely not necessary and imo it doesn't make anything better :man_shrugging:

Reply:
Do you mean using `_ <- ...; pass` instead of `void $` or returning the newly created `HdAccount` from `createAccPrefiltered`?

Reply:
You don't need `void`, `forM_` should discard results for you.

Reply:
`forM_ :: (Foldable t, Monad m) => t a -> (a -> m b) -> m ()`